### PR TITLE
Proposal and tutorial of unstructured lifelong learning

### PR DESCRIPTION
**Pull request by luosiqi**
Proposal of unstructured lifelong learning and describe the following key points:
1. Support unstructured data
2. Add unseen task processing online and offline
3. Change of system architecture

### CODE REVIEW

File: docs/proposals/lifelong-learning/unstructured-lifelong-learning.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+     * [Lifelong Learning Controller](#lifelong-learning-controller)
+     * [Downstream Controller](#downstream-controller)
+     * [Upstream Controller](#upstream-controller)
+     * [Details of api between GM(cloud) and LC(edge)](#details-of-api-between-gmcloud-and-lcedge)
+   * [Workers Communication](#workers-communication)
+
+# Unstructured Lifelong Learning
+## Motivation
+
+Edge cloud synergy lifelong learning has been proposed and realized through Sedna in [ATCII example](https://github.com/kubeedge/sedna/blob/main/examples/lifelong_learning/atcii/README.md) which handles structured data. However, the last version of Sedna lifelong learning scheme has the following limitations.
```
Comment (MooreZheng) [inline docs/proposals/lifelong-learning/unstructured-lifelong-learning.md:23]: Might specify which the mentioned version is.

File: docs/proposals/lifelong-learning/unstructured-lifelong-learning.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+* **It hasn't enable to detect or tackle unseen samples**, also called heterogeneous samples, at the inference stage. As is well known, Non-IID is prevailing situation that exists in distributed AI. When encountering heterogeneous data, the real-time inference performance of original model is greatly reduced which might cause unacceptable loss in certain scenarios. Hence, how to detect heterogeneous or unseen data in advance and process them properly in real time become sigificant topics.
+
+* **It has not realized efficient unseen task processing**. As unseen samples are collected, turning them to be seen samples in order to make model smarter is challenging and important. Efficiently training unseen sample training in addition to retraining all the data is worth studying. 
+
+In this version of lifelong learning, we seek to fundamentally overcome the above challenges of edge-cloud distributed machine learning.
+### Goals
+
+* Supports unstructured lifelong learning.
```
Comment (jaypume) [inline docs/proposals/lifelong-learning/unstructured-lifelong-learning.md:34]: It is not clear for reader to understand, like "the goal of unstructured lifelong learning is to support unstructured lifelong learning". I suggest to give a concrete explain about this goal, at least you can say "Support unstructured dataset like images"

File: docs/proposals/lifelong-learning/structured-lifelong-learning.md [hunk 3]
```diff
@@ -18,12 +18,8 @@
    * [Workers Communication](#workers-communication)
 
 # Lifelong Learning
-## Motivation
-
-
-At present, edge-cloud synergy machine learning is confronted with the challenge of heterogeneous data distributions in complex scenarios and small samples on the edge. The edge-cloud synergy lifelong learning is accordingly proposed: 1) In order to learn with shared knowledge between historical scenarios, the scheme is essentially the combination of another two learning schemes, i.e., multi-task learning and incremental learning; 2) The cloud knowledge base in lifelong learning empowers the scheme with memory ability, which helps to adapt historical knowledge to new and unseen situations on the edge. Joining the forces of multi-task learning, incremental learning and the knowledge base, the lifelong learning scheme seeks to fundamentally overcome the above challenges of edge-cloud synergy machine learning.
 ### Goals
```
Comment (MooreZheng) [inline docs/proposals/lifelong-learning/structured-lifelong-learning.md:21]: More details are needed

Comment (luosiqi) [reply docs/proposals/lifelong-learning/structured-lifelong-learning.md:21]: Supplemented

File: docs/proposals/lifelong-learning/unstructured-lifelong-learning.md (hunk 2 above)
Comment (luosiqi) [reply docs/proposals/lifelong-learning/unstructured-lifelong-learning.md:34]: done

File: docs/proposals/lifelong-learning/unstructured-lifelong-learning.md (hunk 1 above)
Comment (luosiqi) [reply docs/proposals/lifelong-learning/unstructured-lifelong-learning.md:23]: The version has been specified.

**Comment by jaypume** [comment]
Looks good to me, but please squase these commits to make commit history more clear.

**Pull request merged**